src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -37,27 +40,36 @@
 /**
  * @author Peter Güttinger
  */
-@Name("Data Value")
-@Description({"The data value of an item.",
-		"You usually don't need this expression as you can check and set items with aliases easily, " +
-				"but this expression can e.g. be used to \"add 1 to data of &lt;item&gt;\", e.g. for cycling through all wool colours."})
-@Examples({"add 1 to the data value of the clicked block"})
+@Name("Data/Damage Value")
+@Description({"The data/damage value of an item/block. Data values of blocks are only supported on 1.12.2 and below.",
+		"You usually don't need this expression as you can check and set items with aliases easily, ",
+		"but this expression can e.g. be used to \"add 1 to data of &lt;item&gt;\", e.g. for cycling through all wool colours."})
+@Examples({"set damage value of player's tool to 10",
+		"set data value of target block of player to 3",
+		"add 1 to the data value of the clicked block",
+		"reset data value of block at player"})
 @Since("1.2")
 public class ExprDurability extends SimplePropertyExpression<Object, Short> {
```

Comment:
Maybe we could replace this with number? Or that could be for a future PR, up to you :)

Reply:
Good thinking, ill change that up now.